### PR TITLE
1069 change wordings details page

### LINKED ISSUE #1069: Change wordings for capability and contribution view page
Change wordings for capability and contribution view page.
In contribution view page the "contribution" word gets repeated multiple times. Ex: "Contribution name", "contribution short description" . The button names have "contribution" repeated. "Edit contribution" "delete contribution" "review contribution"

### PR DESCRIPTION
**Pull request by minump**
## Description
Wordings for view pages are changed.
Contribution-details, capability-details and talent-details page labels are changed.
"Delete Contribution", "Edit Contribution" and "Review contribution" button texts changed to "Review" "Edit" and "Delete".

Converted some uppercase letters to lowercase. 

**Fixes #1069**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [x] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [x] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

File: contributions/catalog/webapps/templates/contribute/capability_details.html
Comment (sandeep-ps) [inline contributions/catalog/webapps/templates/contribute/capability_details.html:1]: Hi, @minump, I wonder if we need to update the word capitalizations. We are using similar label styles for the Events Manager app. Reducing repeated words like Contribution, Capability, and Talent sounds good. Please share your thoughts. Thanks.

File: contributions/catalog/webapps/templates/contribute/talent_details.html [hunk 1]
```diff
@@ -125,7 +125,7 @@ <h4>Minimum user privacy level</h4>
         {% if post.minEndUserRoles %}
         <div class="row">
             <div class="col-12 col-md-3">
-                <h4>Minimum set of end-user roles</h4>
+                <h4>Minimum Set of End-user Roles</h4>
```
Comment (sandeep-ps) [inline contributions/catalog/webapps/templates/contribute/talent_details.html:128]:
```suggestion
                <h4>End-user Roles</h4>
```

I think, here, "Minimum Set" could be omitted.

File: contributions/catalog/webapps/templates/contribute/capability_details.html
Comment (sandeep-ps) [reply contributions/catalog/webapps/templates/contribute/capability_details.html:1]: Thanks.

**Review by sandeep-ps — APPROVED** [review]
Other changes look good. Approving. Thanks.

**Comment by minump** [comment]
Merging to develop

**Pull request merged**